src/components/battleComponets/DisplayButtons.jsx

```diff
@@ -1,37 +1,23 @@
-function DisplayButtons(props){
-    function answeredWrong(){
-        console.log("forkert");
-        props.setOppLife(props.oppLife-1);
-        console.log(props.oppLife);
-        if (props.oppLife == 1){
-            props.setGameResult("won");
-            props.setShowWonPopUp(true);
-        }
-        // TODO: Lav så den også sender til server om at det var forkert
-    }
-    function answeredRight(){
-        console.log("korrekt");
-        // TODO: Lav så den også sender til server om at det var forkert
-    }
+import { socket } from "../../socket.js"
 
+function DisplayButtons(props){
     if (props.myTurn == true){
         return(
             <div className="row p-2 justify-content-md-center">
                 <div className="col-6 col-md-auto d-grid">
                     <label htmlFor="voteBtn">Was the answer correct or wrong?</label><br/>
-                    <button type="button" className="btn btn-success p-3"id="voteBtn" onClick={()=>answeredRight()}>correct!</button>
-                    <button type="button" className="btn btn-success p-3"id="voteBtn" onClick={()=>answeredWrong()}>wrong!</button>
+                    <button type="button" className="btn btn-success p-3"id="voteBtn" onClick={()=>{socket.emit("answerReview", {value:true});props.setHideElement(true)}}>correct!</button>
+                    <button type="button" className="btn btn-success p-3"id="voteBtn" onClick={()=>{socket.emit("answerReview",{value:false});props.setHideElement(true)}}>wrong!</button>
```

Comment:
For my sanity, please tilføj lidt mellemrum så det er lidt mere læsbart :) Det behøver ikke alt at være så kompakt som muligt, det sørger Vite for at gøre når man bygger til "production" ved `npm run build` :)
```suggestion
                    <button type="button" className="btn btn-success p-3" id="voteBtn" onClick={()=>{socket.emit("answerReview", {value: true}); props.setHideElement(true)}}>correct!</button>
                    <button type="button" className="btn btn-success p-3" id="voteBtn" onClick={()=>{socket.emit("answerReview", {value: false}); props.setHideElement(true)}}>wrong!</button>
```